manifests/alertmanager.pp

```diff
@@ -177,7 +177,7 @@
 
   file { $config_dir:
     ensure  => 'directory',
-    owner   => $user,
+    owner   => 'root',
```

Comment:
Wasn't there an issue on  BSD and this needed to be `0` instead of `root`? I'm not 100% sure anymore.

Reply:
no, that was only the group that is `wheel` instead of `root`, and where you use `gid=0`. this is the `owner`, and `root` definitely exists on bsd.